.github/workflows/push.yml

```diff
@@ -1,18 +1,31 @@
 on: push
 name: Continuous Integration
 jobs:
-  checks:
-    name: Checks
+  test:
     runs-on: ubuntu-latest
+
+    env:
+      MIX_ENV: test
     steps:
-    - uses: actions/checkout@master
-    - name: Get Deps
-      uses: jclem/action-mix/deps.get@v1.3.3
-    - name: Check Credo
-      uses: jclem/action-mix@v1.3.3
-      with:
-        args: credo --strict
-    - name: Check Formatting
-      uses: jclem/action-mix@v1.3.3
-      with:
-        args: format --check-formatted
+      - uses: actions/checkout@v1.0.0
```

Comment:
not v2? https://github.com/actions/checkout

Reply:
hah! good call! Didn't see that release!